addons/block_code/ui/block_canvas/block_canvas.gd

```diff
@@ -92,6 +98,40 @@ func _can_drop_data(at_position: Vector2, data: Variant) -> bool:
 
 
 func _drop_data(at_position: Vector2, data: Variant) -> void:
+	var type = data.get("type", "")
+	if type == "nodes":
+		_drop_node(at_position, data)
+	elif type == "obj_property":
+		_drop_obj_property(at_position, data)
+
+
+func _drop_obj_property(at_position: Vector2, data: Variant) -> void:
+	var obj_name = str(data.get("object", "")).get_slice(":", 0)
+	var prop_name = data.get("property", "")
+	var val = data.get("value", null)
+	if obj_name == "" || prop_name == "" || val == null:
+		return
+
+	# Prepare a block to get the property's value
+	var block_definition = (
+		BlockDefinition
+		. new(
+			&"%s_get_%s" % [obj_name, prop_name],
+			obj_name,
+			"The %s property" % prop_name,
+			"Physics | Velocity",  # TODO: Should get this Category dynamically
```

Comment:
Do we need to set a valid category? The block doesn't get added to the picker. Is this just for the sake of its colour?

Reply:
Yes, for the color.

Reply:
Maybe use the Variables category?

Reply:
Hmm!!! "Variables" sounds a good category!